api/src/main/java/io/druid/data/input/impl/FloatDimensionSchema.java

```diff
@@ -27,10 +27,16 @@
 {
   @JsonCreator
   public FloatDimensionSchema(
-      @JsonProperty("name") String name
+      @JsonProperty("name") String name,
+      @JsonProperty("multiValueHandling") MultiValueHandling multiValueHandling
```

Comment:
We are supposed to make array type metric but it can be done in following PR. consider it done.